core-bundle/src/Controller/AbstractController.php

```diff
@@ -124,4 +124,13 @@ protected function setCacheHeaders(Response $response, PageModel $pageModel): Re
 
         return $response;
     }
+
+    protected function hasParameter(string $name): bool
+    {
+        if (!$this->container->has('parameter_bag')) {
```

Comment:
In which case is this true?

Reply:
It's a service locator, if you pass an entirely custom service locator, you might not pass any parameter bag. The same is done in `getParameter()` in the Symfony AbstractController.

Reply:
The Symfony `AbstractController` makes the same check in its `getParameter` method and would throw an exception with the following message:

```
The "AbstractController::getParameter()" method is missing a parameter bag to work properly. Did you forget to register your controller as a service subscriber? This can be fixed either by using autoconfiguration or by manually wiring a "parameter_bag" in the service locator passed to the controller.
```

Reply:
I see. But Symfony throws an exception in this case and we return `false` instead. Shouldn‘t we do what Symfony does?

Reply:
No, the entire point of this method is to NOT throw an exception :D

Reply:
Fine with me then.